### ISSUE TITLE
"pseudo_set" for train() function

### ISSUE THREAD

**Issue by nutellamok** [chenyaofo/CTNAS#4]
Where do you define the "pseudo_set" for the train() function in line 53 of train.py?
Right now, it appears that you are feeding "None" as the pseudo-set variable:
        accuracy, loss = train(epoch=epoch, labeled_loader=train_loader, pseudo_set=None,
                               pseudo_ratio=args.pseudo_ratio,
                               nac=nac, criterion=criterion, optimizer=nac_optimizer)

**Comment by nutellamok** [comment]
Nvmd, I found it! Thank you for releasing your code :)
